frappe/integrations/doctype/google_settings/google_settings.py

```diff
@@ -2,11 +2,29 @@
 # Copyright (c) 2019, Frappe Technologies and contributors
 # For license information, please see license.txt
 
-# import frappe
+import frappe
 from frappe.model.document import Document
 
 class GoogleSettings(Document):
 	pass
 
 def get_auth_url():
-	return "https://www.googleapis.com/oauth2/v4/token"
\ No newline at end of file
+	return "https://www.googleapis.com/oauth2/v4/token"
+
+
+@frappe.whitelist(allow_guest=True)
```

Comment:
If the `allow_guest` is set to `False`, will that cause issues when trying to make requests on the frontend? Since this is private data being returned, I think we should be extra safe.